Dockerfile

```diff
@@ -1,33 +1,12 @@
 FROM golang:alpine AS builder
 WORKDIR /src
 
-ENV USER=vpn-webauth
-ENV UID=10001 
-
 COPY go.mod .
 COPY go.sum .
 RUN go mod download
 
 RUN apk --update --upgrade --no-cache add git gcc g++ ca-certificates && update-ca-certificates
-RUN adduser \    
-    --disabled-password \    
-    --gecos "" \    
-    --home "/nonexistent" \    
-    --shell "/sbin/nologin" \    
-    --no-create-home \    
-    --uid "${UID}" \    
-    "${USER}"
 
 COPY . .
-RUN CGO_ENABLED=1 GOOS=linux go build -a -installsuffix cgo -ldflags="-w -s" -o /out/vpn-webauth .
-
-
-FROM alpine AS bin
```

Comment:
This probably shouldn't be removed since it creates a minimal image without the packages used during compilation.

Reply:
See previous comment, I edited the Docker container due to issues I was having with getting the templates to update. I'll try again and see if I can post what the issues were.

Reply:
Build is working now